plugin-animation/src/test/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImplTest.kt

```diff
@@ -972,6 +974,60 @@ class CameraAnimationsPluginImplTest {
     unmockkStatic(ShadowLogger::class)
   }
 
+  @Test
+  fun catchErrorOnSetCameraTest() {
```

Comment:
I guess that the differences between this test and `catchExceptionOnSetCameraTest` is only throwing different type of error. 
so maybe they should be merged?

Reply:
Yeah, the only difference is the type of error, but let's keep it just to be sure that both cases pass.

Reply:
guess @alexander-kulikovskii was talking more about introducing private function and reusing the code and not duplicating it.